go.mod

```diff
@@ -1,11 +1,11 @@
 // module github.com/aimoda/terraform-provider-corellium
 module terraform-provider-corellium
 
-go 1.20
+go 1.21
 
 require (
-	github.com/aimoda/go-corellium-api-client v0.0.0-20230416012942-39f2f87fa661
-	github.com/gruntwork-io/terratest v0.41.25
+	github.com/aimoda/go-corellium-api-client v0.0.0-20231002161040-b35e3f0d181c
```

Comment:
Could we switch to github.com/corellium/go-corellium-api-client instead?

Reply:
Done